pkgs/development/libraries/libuna/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, lib, fetchurl, autoconf, automake, libtool, dos2unix, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  name = "libuna";
+
+  src = fetchurl {
+    url = "https://github.com/libyal/libuna/archive/20150927.tar.gz";
+    sha256 = "0dyvsrbwy6hd6yli57rqpl4q1kfvvs40fvrm4ngna8fnhgvykvlb";
+  };
+
+  buildInputs = [ autoconf automake libtool dos2unix ];
+  nativeBuildInputs = [ pkgconfig ];
```

Comment:
Not that it matters, but autotools are typically included in the native build tools as well.